app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    sum_of_ratings = 0
+    for player in players:
+        sum_of_ratings += player.get_rating()
+    return sum_of_ratings
```

Comment:
Implement this method in one line using `sum` according [to the checklist](https://github.com/Fedot0v/py-elves-and-dwarves/blob/develop/checklist.md#code-efficiency).